src/GraphMetadataViewExtension/Controls/CustomPropertyControl.xaml.cs

```diff
@@ -19,19 +20,36 @@ namespace Dynamo.GraphMetadata.Controls
     /// <summary>
     /// Interaction logic for CustomPropertyControl.xaml
     /// </summary>
-    public partial class CustomPropertyControl : UserControl
+    public partial class CustomPropertyControl : UserControl, INotifyPropertyChanged
     {
-        public DelegateCommand EditPropertyNameCmd;
-        public DelegateCommand DeletePropertyNameCmd;
+        public DelegateCommand EditPropertyNameCmd { get; set; }
+        public DelegateCommand DeletePropertyNameCmd { get; set; }
         public bool PropertyNameEnabled { get; set; }
 
+        /// <summary>
+        /// Event raised when an instance of this class requires deleting.
+        /// </summary>
+        public event EventHandler RequestDelete;
+
+        public void DeleteRequested(EventArgs e)
+        {
+            EventHandler handler = RequestDelete;
+            handler?.Invoke(this, e);
```

Comment:
So this will change to:
`RequestDelete?.Invoke(this, e);`

Reply:
Done.